deploying/rhoaiserving-custom/serving_runtime.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+labels:
+  opendatahub.io/dashboard: "true"
+metadata:
+  name: kserve-sklearnserver
+spec:
+  annotations:
+    prometheus.kserve.io/port: '8080'
+    prometheus.kserve.io/path: "/metrics"
+    openshift.io/display-name: "My Scikit-learn server"
```

Comment:
Note that the correct place for this display-name is before, in the annotations section. If you change it to the correct place, the RHOAI dashboard will show "My Scikit-learn server" in the select box.
RHOAI takes the `ServingRuntime` object, and creates a template for it in `redhat-ods-applications`.
You can compare the generated template with the templates in `oc get template -n redhat-ods-applications`. 
Also, I don't know why in the RHOAI dashboard, the custom server runtime shows "Not defined" as "API protocol".